modules/core/src/test/java/org/apache/ignite/internal/processors/cache/mvcc/CacheMvccConfigurationValidationTest.java

```diff
@@ -352,6 +353,37 @@ public void testMvccEnabledForClientMode() throws Exception {
         assertTrue(node.context().coordinators().mvccEnabled());
     }
 
+    /**
+     * Check that node in client mode (filtered by AttributeNodeFilter) correctly works with MVCC.
+     */
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testMvccEnabledForClientModeAfterTriggeringGlobalState() throws Exception {
```

Comment:
looks like a 90% duplication of testMvccEnabledForClientMode

Reply:
Move to another class to move some code duplication, and add more tests